### PR TITLE
Fix Documentation Link and Missing Kafka References

### PR DESCRIPTION
**Pull request by Jack-Kilrain**
Updates to the `README.md`:
* Fixed contributing doc link
* Fixed Redis example link
* Adding missing reference to Kafka in examples

### CODE REVIEW

**Comment by Jack-Kilrain** [comment]
Need some confirmation on the Kafka entry, has this intentionally been left out for now?

File: README.md [hunk 1]
```diff
@@ -23,17 +23,20 @@ Shotover currently supports intercepting requests for the following technologies
...
 * Cassandra
 * Redis
+* Kafka
 
 ## What problems does Shotover solve?
 
 Concrete examples where Shotover has been applied include:
 
-* [Multi-region, active-active redis](shotover-proxy/tests/test-configs/redis-multi/)
+* [Multi-region, active-active redis](shotover-proxy/tests/test-configs/redis/multi)
```
Comment (rukai) [inline README.md:38]: While your here can you make this point at https://github.com/shotover/shotover-examples/tree/main/redis-backup-cluster instead?

File: README.md [hunk 2]
```diff
@@ -23,17 +23,20 @@ Shotover currently supports intercepting requests for the following technologies
...
+* [Multi-region Kafka](shotover-proxy/tests/test-configs/kafka/cluster/)
```
Comment (rukai) [inline README.md:39]: We dont actually deploy shotover kafka anywhere, which is the point of this section, so lets not mention it here.

**Review by rukai — APPROVED** [review]
Thanks!
Kafka has been long undocumented because it was considered alpha stage.
But things have stabilized so we should really remove it from alpha-transforms feature flag and start documenting it.

File: README.md (hunk 2 above)
Comment (Jack-Kilrain) [reply README.md:39]: Ah ok gotcha. I'll nuke this bit.

File: README.md (hunk 1 above)
Comment (Jack-Kilrain) [reply README.md:38]: Yep, sounds good, I'll add it.

**Review by conorbros — APPROVED** [review]

**Pull request merged**